src/stories/Editor.js

```diff
@@ -0,0 +1,157 @@
+import Underline from '@ckeditor/ckeditor5-basic-styles/src/underline';
+import Widget from '@ckeditor/ckeditor5-widget/src/widget';
+import { add } from '@ckeditor/ckeditor5-utils/src/translation-service';
+import { getEnvKeystrokeText } from '@ckeditor/ckeditor5-utils/src/keyboard';
+
+/**
+ * Garden theme customizations
+ */
+import '../theme/theme.css';
+
+/**
+ * Add custom tooltip values
+ */
+add('en', {
+  'Bulleted List': `Bulleted list (${getEnvKeystrokeText('CTRL+SHIFT+8')})`,
+  'Numbered List': `Numbered list (${getEnvKeystrokeText('CTRL+SHIFT+7')})`,
+  'Decrease indent': `Decrease indent (${getEnvKeystrokeText('CTRL+[')})`,
+  'Increase indent': `Increase indent (${getEnvKeystrokeText('CTRL+]')})`,
+  'Block quote': `Blockquote (${getEnvKeystrokeText('CTRL+SHIFT+9')})`,
+  Code: `Code (${getEnvKeystrokeText('CTRL+SHIFT+5')})`,
+  'Horizontal line': `Horizontal rule (${getEnvKeystrokeText('CTRL+SHIFT+L')})`
+});
+
+const plugins = [
+  Essentials,
+  Bold,
+  Italic,
+  Paragraph,
+  Underline,
+  Code,
+  HorizontalLine,
+  Link,
+  Indent,
+  IndentBlock,
+  List,
+  BlockQuote,
+  Heading,
+  CodeBlockEditing,
+  CodeBlockUI,
+  Widget
+];
+
+const toolbar = [
+  'heading',
+  'bold',
+  'italic',
+  'underline',
+  'code',
+  '|',
+  'bulletedList',
+  'numberedList',
+  'outdent',
+  'indent',
+  '|',
+  'blockquote',
+  'codeBlock',
+  'link',
+  'horizontalLine'
+];
+
+const codeBlock = {
+  languages: [{ language: 'plaintext', label: 'Plain text' }]
+};
+
+export const Editor = ({ data, onChange, toolbarClassName }) => {
+  const onReady = editor => {
+    /**
+     * Apply custom keyboard shortcuts
+     */
+    editor.keystrokes.set('CTRL+SHIFT+8', 'bulletedList');
+    editor.keystrokes.set('CTRL+SHIFT+7', 'numberedList');
+    editor.keystrokes.set(['ctrl', 219], 'outdent'); // CTRL+[
+    editor.keystrokes.set(['ctrl', 221], 'indent'); // CTRL+]
+    editor.keystrokes.set('CTRL+SHIFT+9', 'blockQuote');
+    editor.keystrokes.set('CTRL+SHIFT+5', 'code');
+    editor.keystrokes.set('CTRL+SHIFT+L', 'horizontalLine');
+
+    /**
+     * Attach decoupled toolbar
+     */
+    const toolbarElement = document.querySelector(`.${toolbarClassName}`);
+
+    toolbarElement.appendChild(editor.ui.view.toolbar.element);
+
+    /**
+     * Reset tooltip placement
+     */
+    const toolbarItems = editor.ui.view.toolbar.items;
+
+    for (const toolbarItem of toolbarItems) {
+      if (toolbarItem.buttonView) {
+        toolbarItem.buttonView.set('tooltipPosition', 'n');
+      } else {
+        toolbarItem.set('tooltipPosition', 'n');
+      }
+    }
+
+    /**
+     * Reset overflow dropdown placement
+     */
+    if (!editor.config.get('toolbar').shouldNotGroupWhenFull) {
+      const overflowDropdown = editor.ui.view.toolbar._behavior.groupedItemsDropdown;
+
+      overflowDropdown.set('panelPosition', 'nw');
+      overflowDropdown.buttonView.set('tooltipPosition', 'n');
+    }
```

Comment:
@hzhu these were the lines suggested by CK to fix the comments made by @allisonacs [here](https://github.com/zendeskgarden/ckeditor/pull/11#issuecomment-845423494) and [here](https://github.com/zendeskgarden/ckeditor/pull/11#issuecomment-845424147)